presentation-api/receiving-ua/support/PresentationReceiver_create_receiving-ua_child.html

```diff
@@ -61,31 +61,43 @@
 
         // Indexed Database
         let db;
+        const storeName = 'store-controlling-ua';
         const checkIndexedDB = () => {
             if ('indexedDB' in window) {
                 message = 'Indexed Database is shared by top-level and nested browsing contexts.';
```

Comment:
Isn't this required by same-origin policy?  Do we need to test this here?

Reply:
The intention of this is to check if both top-level and nested browsing contexts share the same storage, according to the following spec described in [Creating a receiving browsing context](http://w3c.github.io/presentation-api/#creating-a-receiving-browsing-context):

> All nested browsing contexts created by the presented document, i.e. that have the receiving browsing context as their top-level browsing context, [...] MUST also share the same browsing state (storage) for features 5-10 listed above.